src/datasource.ts

```diff
@@ -588,6 +802,9 @@ export async function getDataQueryRequestItems(
 }
 
 function groupTargets(targets: CogniteQuery[]) {
-  const [eventTargets, tsTargets] = partition(targets, ({ tab }) => tab === Tab.Event);
+  const [eventTargets, tsTargets] = partition(
+    targets,
+    ({ tab }) => tab === Tab.Event || tab === Tab.Relationships
```

Comment:
this function needs to be rewritten to support 3 groups, something like:

```
const { events, timeseries, relationships } = groupBy(targets,  ({ tab }) => (tab || Tab.Timesseries).toLowerCase() ))
return { eventTargets: events, tsTargets: timeseries, relationshipsTargets: relationships };
```

Reply:
fyi `Tab.Timesseries` is the default, so undefined should appear as timeseries as well

Reply:
>1. ` case Tab.Relationships: { return relationsShipsQuery; }`
here I can't return a bool value. I have to return the object holding the selected labels and datasets by default are empty arrays see `defaultRelationshipsQuery: RelationshipsQuery`

>2. for `const [eventTargets, tsTargets] = partition(targets, ({ tab }) => tab === Tab.Event);` see next comment with: `fetchEventTargets && groupTargets`

Reply:
>  here I can't return a bool value. I have to return the object holding the selected labels and datasets by default are empty arrays see defaultRelationshipsQuery: RelationshipsQuery

could you elaborate? I don't see the reasoning

Reply:
I fixed it, I've put a constant true and now it works!!!